.github/workflows/build_v2.yml

```diff
@@ -0,0 +1,91 @@
+name: build-v2
+
+on: push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: v2
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      
+      - name: Cache Node modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}-v2
+
+      - name: Install dependencies
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: yarn install
+
+      - name: Synthesize project files
+        run: npx projen
+
+      - name: Anti-tamper check
+        run: |-
+          if ! git diff --exit-code; then
+            echo "Do not directly edit the project files generated by Projen. Add your changes to '.projenrc.js' then run 'npx projen'."
+            exit 1
+          fi
+
+      - name: Check formatting
+        run: yarn check-formatting
+
+      - name: Lint
+        run: yarn eslint
+
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: v2
+    strategy:
+      max-parallel: 4
+      matrix:
+        node-version: [14,16]
```

Comment:
The reason 12 isnt in here is because of this [convo](https://dd.slack.com/archives/GHQ114DRR/p1643308960053100) right?

Reply:
yep!